openedx/features/edly/utils.py

```diff
@@ -26,6 +26,20 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def is_edly_sub_org_active(edly_sub_org):
+    """
+    Checks if the request EdlySubOrganization is enabled or disabled.
+
+    Arguments:
+        request: EdlySubOrganization model class
+
+    Returns:
+        bool: Returns True if site is enabled and False if the site is disabled or DoesNotExist
+    """
```

Comment:
if it `DoesNotExist`, this method will break.

Reply:
The function is only called if the EdlySubOrganization exists.